client/src/components/ContentNavigator/ContentModel.ts

```diff
@@ -244,16 +227,13 @@ export class ContentModel {
   ): Promise<ContentItem | undefined> {
     const contentType = await this.getFileContentType(fileName);
```

Comment:
nit: it looks a little confusing with `fetchFileContentType`. It's hard to tell the difference from the name "fetch" vs. "get". The implementation look unrelated to each other.

Reply:
I updated the method to better match what is being returned, and updating both "fetch" methods to use a prefix of "get" since that seems to be more consistent with what we use elsewhere https://github.com/sassoftware/vscode-sas-extension/pull/980/commits/1ee77cc65681082fc3b202e55e265821d23a11db